packages/ide/src/client.ts

```diff
@@ -35,10 +35,13 @@ export abstract class IdeClient {
 
 		let appWindow: Window | undefined;
 
-		window.addEventListener("beforeunload", (e) => {
-			e.preventDefault(); // FireFox
-			e.returnValue = ""; // Chrome
-		});
+		if (!window.matchMedia("(display-mode: standalone)").matches) {
```

Comment:
Won't match fullscreen. Is there a cleaner way to check if we're in a minified window mode?

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/display-mode#Syntax

Reply:
The fullscreen mode would still have the problem where `control + w` closing the browser window, correct?

Reply:
`standalone` should cover it afaik